lib/chef/win32/api/file.rb

```diff
@@ -565,6 +600,21 @@ def retrieve_file_info(file_name)
           file_information
         end
 
+        def retrieve_file_version_info(file_name)
+          file_name = encode_path(file_name)
+          file_size = GetFileVersionInfoSizeW(file_name, "")
+          if file_size == 0
+            Chef::ReservedNames::Win32::Error.raise!
```

Comment:
I wonder if it makes sense to call `Chef::ReservedNames::Win32::Error.get_last_error` here and print that in the exception. I thought we did that but apparently the only place we reference it is in some of the file security code.